foundations/src/telemetry/log/internal.rs

```diff
@@ -36,7 +91,6 @@ pub fn current_log() -> SharedLog {
 
 pub(crate) fn fork_log() -> SharedLog {
     let parent = current_log();
-    let log = parent.read().new(slog::o!());
-
-    Arc::new(parking_lot::RwLock::new(log))
+    let log_lock = parent.read();
```

Comment:
I'm not sure why it wouldn't be functionally the same. Calling `Logger::new()` with an empty KV will clone the drain and add zero fields to the KV. Which is basically what `<LogContext as Clone>::clone` will do, too.

I'm open to being convinced otherwise! but I will leave this as is for now.

Reply:
Ah - I mistakenly thought `o!` was making a new output or some such, not the KV. Looking at slog's source now, I agree, I think you're right and what you're doing is functionally the same.